cext/src/_twisted_c_extensions/_setup.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+# pylint: disable=I0011,C0103,C9302,W9401,W9402
+
+"""
+Setuptools convenience functionality.
+
+This file must not import anything from Twisted, as it is loaded by C{exec} in
+C{setup.py}. If you need compatibility functions for this code, duplicate them
```

Comment:
"duplicate them here" <- this seems like a recipe for a nightmare of copy/pasting.